30-remote-site-setup.sh

```diff
@@ -35,6 +35,37 @@ EOF
   ls -l "$ssh_dir"
 }
 
+# Populate the bosco override dir from a Git repo
+# Expected Git repo layout:
+#     RESOURCE_NAME_1/
+#         bosco_override/
+#         ...
+#     RESOURCE_NAME_2/
+#         bosco_override/
+#         ...
+#     ...
+function bosco_override_setup {
+    GIT_ENDPOINT=$1
+    RESOURCE_NAME=$2
+
+    # Set up config for SSH endpoints
+    if [[ $GIT_ENDPOINT =~ ^[A-Za-z0-9_-]+@([^:]+): ]]; then
+        ssh-keyscan "${BASH_REMATCH[1]}" >> ~/.ssh/known_hosts
+    fi
+
+    REPO_DIR=$(mktemp -d)
+    OVERRIDE_DIR=/etc/condor-ce/bosco_override/
+
+    git clone --depth=1 $GIT_ENDPOINT $REPO_DIR
+
+    # Bosco override dirs are expected in the following location in the git repo:
+    #   <RESOURCE NAME>/bosco_override/
+    RESOURCE_DIR="$REPO_DIR/$RESOURCE_NAME/"
+    [[ -d $RESOURCE_DIR ]] || errexit "Could not find $RESOURCE_NAME/ under $GIT_ENDPOINT"
+    rsync -az "$RESOURCE_DIR/bosco_override/"  $OVERRIDE_DIR
```

Comment:
```suggestion
    rsync -a "$RESOURCE_DIR/bosco_override/"  $OVERRIDE_DIR
```
No reason to compress and uncompress if you're not copying between hosts.